rules/lib/is-promise.js

```diff
@@ -2,37 +2,35 @@
  * Library: isPromise
  * Makes sure that an Expression node is part of a promise.
  */
-var STATIC_METHODS = [
-  'all',
-  'race',
-  'reject',
-  'resolve'
-]
+
+'use strict'
+
+const PROMISE_STATICS = require('./promise-statics')
```

Comment:
Open to a better file/module name than `promise-statics` if you have suggestions!